ens-improvement-proposals/ensip-xx-ccip-metadata.md

```diff
@@ -30,10 +30,35 @@ The metadata should include 2 different types of info
 ```solidity
 
 interface OffChainResolver {
-    owner(bytes32 node) returns (bytes32); // first 20 bytes are assumed to be the address if it's an evm chain, but gives space for longer address on non-evm chains.
+    /** @dev Returns the owner of the resolver on L2
+     * @param node
+     * @return owner gives space for longer address on non-evm chains.
```

Comment:
I think it would be better to have more clear description here, maybe example of returned data for both cases